src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -2104,6 +2128,24 @@ def get_variable_OM_costs(
         # assume the user is not using this
         b.other_variable_costs.fix(0)
 
+        # variable for user to assign watertap variable costs to,
+        # constraint sets to sum of list, which is 0 for empty list
+        b.watertap_variable_costs = Var(
+            initialize=0,
+            bounds=(0, 1e4),
```

Comment:
Is there a reason an upper bound of 1e4 is used instead of None?

Reply:
@andrewlee94 and I have been discussing removing the bounds for other reasons, as the upper limit is valid for UKy in millions of USD but not necessarily other case studies or currencies. Maybe this PR is a good place to just remove all of the bounds. Do you have a suggestion either way, @MarcusHolly?

Reply:
Unless I'm missing something, I don't think there's a good reason to have upper bounds on costs. So I would agree they should all be removed